src/main/java/io/appform/dropwizard/discovery/bundle/Constants.java

```diff
@@ -26,8 +26,9 @@
  */
 @UtilityClass
 public class Constants {
+
     public static final String DEFAULT_NAMESPACE = "default";
-    public static final String DEFAULT_HOST = "__DEFAULT_SERVICE_HOST";
+    public static final String LOCALHOST = "localhost";
```

Comment:
This doesn't look right. __DEFAULT_SERVICE_HOST was set as a marker to ensure that host was not provided, so do a lookup. Localhost here is almost never the correct answer.